android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -99,12 +135,24 @@ public void getPermissionsStatus(final Promise promise) {
             return;
         }
 
-        boolean foreground = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION) == PackageManager.PERMISSION_GRANTED;
+        int gpsLocation = ActivityCompat.checkSelfPermission(activity, Manifest.permission.ACCESS_FINE_LOCATION);
```

Comment:
how confident are we that `FINE` is synonymous with gps and `COARSE` is synonymous with wifi?  I do see some content out there claiming this.

Reply:
wondering if it might be more straightforward to prefix the variables with `fine` and `coarse` instead